### PR TITLE
PKG-1724

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**
it is a dependency for python-isal

### CODE REVIEW

**Review by ELundby45 — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,39 +14,39 @@ source:
...
-        # Test compression and decompression. 
     - igzip -k LICENSE  # [unix]
-        # Original file should be larger than compressed file.
     - bash -c '[ $(stat -c %s LICENSE) -gt $(stat -c %s LICENSE.gz) ]'  # [unix]
     - bash -c '[ "$(igzip -cd LICENSE.gz)" == "$(cat LICENSE)" ]'  # [unix]
 
 about:
-  home: https://software.intel.com/content/www/us/en/develop/tools/isa-l.html
-  license: BSD-3-Clause
+  home: https://www.intel.com/content/www/us/en/develop/tools/isa-l.html
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: The correct link is https://www.intel.com/content/www/us/en/developer/tools/isa-l/overview.html

File: recipe/meta.yaml [hunk 2]
```diff
@@ -14,39 +14,39 @@ source:
     - nasm-filter.patch
 
 build:
-  number: 4
-
+  number: 0
+  skip: true  # [not (x86_64)]
 
 requirements:
+  host:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Could you place an empty `host` section below the `build` section?

File: recipe/meta.yaml (hunk 1 above)
Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml]: done

File: recipe/meta.yaml (hunk 2 above)
Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml]: done

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**